packages/api/__tests__/specs/__snapshots__/peer.ts.snap

```diff
@@ -135,6 +137,8 @@ exports[`Peers can be created/edited/deleted: can update peer profile 1`] = `
 Object {
   "data": Object {
     "updatePeerProfile": Object {
+      "callToActionImage": null,
+      "callToActionImageURL": null,
```

Comment:
here it's `null` and in the test snapshot above it's `""`. I think it should be the same in both.